src/main/java/net/fabricmc/loader/launch/knot/Knot.java

```diff
@@ -0,0 +1,433 @@
+/*
+ * Copyright 2016 FabricMC
```

Comment:
Might want to update the license header to use the current year instead of 2016, or "2016-$current_year", or something similar

Reply:
Separate issue?

Reply:
Could come along with the reformat